### PR TITLE
X-Bot from Baris@dmbtechdev

### PR DESCRIPTION
**Pull request by dmbtechdev**
https://github.com/dmbtechdev/x-bot/blob/master/README-DMB.md

### CODE REVIEW

**Comment by dmbtechdev** [comment]
@mjovanc Hi, would a maintiner approve the warkflow? I run it in fork repo and passed. thx

File: README-DMB.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+Link: [Commit link]
+```
+
+2. New Release Message:
+```
+New release ([Version Number]) of Delta out! 🎉
+Link to release notes: [Release link]
+```
+
+## Setup
```
Comment (mjovanc) [inline README-DMB.md:27]: Can you move this information (all setup information) to the `README.md` file? Then delete this one.

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+axum = "0.7.9"
+serde = { version = "1.0.216", features = ["derive"] }
+serde_json = "1.0.133"
+tracing = "0.1.41"
+tracing-subscriber = { version = "0.3.19", features = ["env-filter"] }
+dotenv = "0.15.0"
+hmac = "0.12.1"
+sha2 = "0.10.8"
+anyhow = "1.0.94"
+octocrab = "0.42.1"        # GitHub API client
```
Comment (mjovanc) [inline Cargo.toml:19]: Remove comments in this file.

Comment (dmbtechdev) [reply Cargo.toml]: with some more clean-ups

**Comment by dmbtechdev** [comment]
> Overall good, thanks a lot for your effort here. Will do some more testing on it. Would you be open to maintaining this one since you basically wrote everything? :)

As long as I could afford, I would and also be happy to contribute. I'd like to collaborate and would be better with more contributors too. Hope you get my DM at X.

**Pull request merged**